Core/combobox.lua

```diff
@@ -581,18 +649,20 @@ dgsRenderer["dgs-dxcombobox"] = function(source,x,y,w,h,mx,my,cx,cy,enabled,eleD
 	dxSetShaderValue(shader,"ocolor",{r/255,g/255,b/255,a/255*parentAlpha})
 	dxDrawImage(x+textBoxLen,y,buttonLen,h,shader,0,0,0,white,isPostGUI)
 	if textBox and not captionEdit then
-		local itemTextPadding = eleData.textPadding
-		local font = eleData.font or systemFont
-		local textColor = eleData.textColor
+		local data = eleData.itemData[eleData.select] or {}
+		local itemTextPadding = dgsElementData[source].itemTextPadding
+		local font = data[-4] or eleData.font or systemFont
+		local textColor = data[-2] or eleData.textColor
 		local rb = eleData.alignment
-		local txtSizX,txtSizY = eleData.textSize[1],eleData.textSize[2] or eleData.textSize[1]
-		local colorcoded = eleData.colorcoded
+		local txtSizX,txtSizY = data[-3] and data[-3][1] or eleData.textSize[1],data[-3] and (data[-3][2] or data[-3][1]) or eleData.textSize[2] or eleData.textSize[1]
+		local colorcoded = data[-5] or eleData.colorcoded
 		local shadow = eleData.shadow
 		local wordbreak = eleData.wordbreak
-		local selection = eleData.select
-		local itemData = eleData.itemData
-		local sele = itemData[selection]
-		local text = sele and sele[1] or eleData.caption
+		local text = data[1] or eleData.caption
+		local image = data[-6]
+		if image and isElement(image[1]) then
+			dxDrawImage(x+image[3],y+image[4],image[5],image[6],image[1],0,0,0,image[2],isPostGUI)
```

Comment:
the height here is different than the one dropbox `itemHeight`
so the image position will be a bit different 
just like in this picture
![mta-screen_2020-12-07_22-40-41](https://user-images.githubusercontent.com/30420446/101400916-9ea04500-38e2-11eb-81dc-7156e45f3a88.png)
_the y here it's not like the one in dropbox_

I tried to do some math but didn't work for me so I left like this